README.md

```diff
@@ -1,59 +1,39 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Leaderboard
 
-# Project Name
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-> One paragraph statement about the project.
+# Leaderboard
 
-![screenshot](./app_screenshot.png)
+![screenshot](./capture.PNG)
 
 Additional description about the project and its features.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML5
+- CSS3
+- Webpack
+- JavaScript
 
-## Live Demo
+<!-- ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://redwing555.github.io/todo-list/) -->
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
```

Comment:
* I think you can modify your readme file to include instructions on how to properly install and run the app this would enable users to set up and use your app without many troubles